src/index.js

```diff
@@ -2,10 +2,20 @@ import AccountData from './AccountData.js'
 import ContractData from './ContractData.js'
 import ContractForm from './ContractForm.js'
 import LoadingContainer from './LoadingContainer.js'
+import AccountDataNew from './react-16.3/AccountData'
+import ContractDataNew from './react-16.3/ContractData'
+import ContractFormNew from './react-16.3/ContractForm'
```

Comment:
Not sure if naming the folder `react-16.3` is quite clear that you are referring to the new context API. Not everyone keeps track of what features are added in what version.

Reply:
Also, I kind of feel like we should release a v2 and name the old way of doing things as legacy, like

```
import X from "legacy/X"
```

Reply:
Hmm yea that's also a possibility. My original concern with that approach is that all the bugfixes in open PRs right now will all have conflicts and it will be that much harder to get them in. 

My thinking was that we get this new api in, then resolve a few large outstanding bugs, then do a 2.0 release with the legacy approach you just suggested.

Thoughts?

Reply:
Sure, but maybe call the folder `new-context-api` instead of `react-16.3`?

Reply:
K will update the name to make it more clear.

Reply:
K will do